app/javascript/gobierto_investments/webapp/mixins/common.js

```diff
@@ -89,6 +89,7 @@ export const CommonsMixin = {
         availableProjectFields
       } = CONFIGURATION;
       const { id: locationId, ...restLocationOptions } = location;
+      const phase_terms = this.convertToArrayOfIds(attributes[phases.id]).map(phase_id => this.middleware.getAttributesByKey(phases.id).vocabulary_terms.find(({ id }) => id === phase_id));
```

Comment:
This could fail. Better split it into several steps:

```suggestion
      const phase_terms = this.convertToArrayOfIds(attributes[phases.id]).map(phase_id => {
        const { vocabulary_terms= [] } = this.middleware.getAttributesByKey(phases.id) || {}
        return vocabulary_terms.find(({ id }) => id === phase_id) || attributes[phases.id]
      }) || [];
```

The last `... || attributes[phases.id]` clause is about the case there wouldn't be a matching vocabulary_term (perhaps a wrong ID), so you return the original object at least